site/topics/selection/selection.rst

```diff
@@ -109,6 +111,124 @@ Tournament Selection
 Fitness Proportional Selection
 ==============================
 
+* There are a collection of selection strategies that are fitness proportaional
+
+    * The probability of selecting an indiviual depends on its fitness value compared to the whole populations's fitness
+
+
+* The simplest of these is to have the probability be the individual's proportion of the total population fitness
+
+    * :math:`p(i) = \frac{f(i)}{\sum^{\mu}_{j=1}f(j)}`
+    * Where :math:`p(i)` is the probabability of selecting individual :math:`i`
+    * And :math:`f(i)` is the fitness of individual :math:`i`
+    * Note that the sum of the probabilities must be one :math:`\sum^{\mu}_{j=1}p(j) = 1`
```

Comment:
Minor, I personally prefer saying "sum over _j_ of blah" rather than specifying _j=1_. _j_ isn't necessarily a number, so saying _j_ only is more... uh... technically correct. 

MORE INCLUSIVE. Go with that one.